demisto_sdk/commands/generate_docs/generate_readme_template.py

```diff
@@ -0,0 +1,16 @@
+import logging
+from pathlib import Path
+
+from demisto_sdk.commands.generate_docs.readme_templates import README_TEMPLATES
+
+logger = logging.getLogger("demisto-sdk")
+
+
+def generate_readme_template(input_path: Path, readme_template: str):
+    with open(input_path, "a") as file_object:
+        template = README_TEMPLATES.get(readme_template, "")
+        if not template:
+            raise Exception(
+                f"[red]Template type {readme_template} is not supported.[/red]"
```

Comment:
Please mention the supported ones (maybe use the constant to do it automatically), so users know what to do